src/developer/web-api/tracker.md

```diff
@@ -2227,6 +2227,8 @@ An example of a json response:
 The change log type can be `CREATE`, `UPDATE`, or `DELETE`.
 `CREATE` and `DELETE` will always hold a single value: the former shows the current value, and the latter shows the value that was deleted. UPDATE will hold two values: the previous and the current.
 
+Change logs are enabled by default and can be configured in `dhis.conf`, as explained [here](https://docs.dhis2.org/en/manage/reference/logging.html#install_changelog).
```

Comment:
Use anchors so the links always link to the doc in the appropriate version. Right now 42 would link to the docs on master once the 42 branch is created.